### PR TITLE
Document voting procedure better in the post that the bot makes on the PR comment threads

### PR DESCRIPTION
**Pull request by rdancer**
Document voting procedure better in the post that the bot makes on the PR comment threads

* re-phrase the post so that people don't question whether the bot only starts counting votes *after* it posts
* describe in detail how votes are tallied
* dissuade cleverheads such as myself from casting duplicate votes

### CODE REVIEW

**Comment by rdancer** [comment]
:+1: PR doesn't count as a vote, so here I go!

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting has begun.

To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must star this repo for your vote to count.**

A decision will be made after this PR has been open for **30** minutes, and at least **5** votes have been made.

*NOTE: the PR will be closed if any new commits are added after:* 8eaa3ccbcdfd5dafb5e802b5eb6b139f8e82d496

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting has begun.

To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must star this repo for your vote to count.**

A decision will be made after this PR has been open for **30** minutes, and at least **5** votes have been made.

A supermajority of 65% is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* 8eaa3ccbcdfd5dafb5e802b5eb6b139f8e82d496

File: voting.js [hunk 1]
```diff
@@ -12,9 +12,14 @@ var decideVoteResult = function(yeas, nays) {
...
 }
 
-var voteStartedComment = '#### :ballot_box_with_check: Voting has begun.\n\n' +
+var voteStartedComment = '#### :ballotbox: Voting procedure reminder:\n' +
   'To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).\n' +
   'Remember, you **must star this repo for your vote to count.**\n\n' +
+  'All comments within this discussion are searched for votes, regardless of the time of posting.\n' +
+  'You can cast as many votes as you want, but only the last one will be counted.\n' +
+  '(You may consider editing your comment instead of adding a new one.)\n' +
+  'Comments containing both up- and dow-votes are disregarded.\n' +
```
Comment (mbertheau) [inline voting.js]: dowN-votes

**Comment by johnnye** [comment]
:+1:

**Comment by clay-to-n** [comment]
:+1:

**Comment by mstoiber** [comment]
:+1:

**Comment by helb** [comment]
:+1:

**Comment by botwillacceptanything** [comment]
#### <a target="_blank" href="http://thecatapi.com/?id=79n"></a>
:+1: The vote passed! This PR will now be merged into master.

----
**Tallies:**
:+1:: 5 (100%) 
:-1:: 0 (0%)

**Pull request merged**